test/extended/authorization/authorization_rbac_proxy.go

```diff
@@ -639,7 +639,8 @@ func RunLegacyEndpointConfirmNoEscalation(t g.GinkgoTInterface, clusterAdminAuth
 			Name: userName,
 		},
 	}
-	escalationFormat := `%s %q is forbidden: user %q (groups=["system:authenticated:oauth" "system:authenticated"]) is attempting to grant RBAC permissions not currently held:`
+	// the two groups appear in different order in different cases
+	escalationFormatRegex := `^%s %q is forbidden: user %q \(groups=\[("system:authenticated" "system:authenticated:oauth"|"system:authenticated:oauth" "system:authenticated")\]\) is attempting to grant RBAC permissions not currently held:`
```

Comment:
nit: another way of doing this would be to extract the groups from the string into a slice and compare. This works, though!